py3status/modules/xscreensaver.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle and lock Xscreensaver.
+
+This is a partial rewrite of dpms.py. Most credit goes to the original author
+Andre Doser. This module shows and controls the status of Xscreensaver
+
+This script is useful for people who let Xscreensaver manage dpms settings.
+Xscreensaver has its own dpms variables separate from xset. Dpms can be
+safely turned off in xset as long as xscreensaver is running.
+Settings can be managed using "xscreensaver-demo".
+
+Configuration parameters:
+    format: default format for this module (default '{output}')
+    format_off: display when Xscreensaver is not running (default '')
+    format_on: display when Xscreensaver is running (default '')
+
+ Format placeholders:
+    {output} display xscreensaver output
+
+@author neutronst4r <c7420{at}gmx{dot}net>
+"""
+from os import setpgrp
+from subprocess import Popen, DEVNULL
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    format = '{output}'
+    format_off = u''
+    format_on = u''
+
+    def post_config_hook(self):
+        if not self.py3.check_commands(['xscreensaver']):
+            self.py3.notify_user('xscreensaver is not installed. cannot use this module.')
+
+    def xscreensaver(self):
+        """
+        """
+        response = {
+            'full_text': self.format_on,
+            'color': self.py3.COLOR_GOOD
+        }
+
+        try:
+            self.run = self.py3.command_output(['pidof', 'xscreensaver'])
```

Comment:
why set `self.run`?  there is no need to store this.  You just care about the Exception